src/corehost/cli/deps_format.cpp

```diff
@@ -69,6 +69,7 @@ void deps_json_t::reconcile_libraries_with_targets(
 
         pal::string_t library_path = get_optional_path(properties, _X("path"));
         pal::string_t library_hash_path = get_optional_path(properties, _X("hashPath"));
+        pal::string_t filter_project_list = get_optional_path(properties, _X("runtimeStoreManifestName"));
```

Comment:
I think it would make sense to keep a consistent name here as what it is called in the deps.json file.  It gets too confusing when we rename the same concept for no reason.